src/pages/default/credentials/AddCredentialDetail.tsx

```diff
@@ -0,0 +1,128 @@
+import { Button } from "@/components/Button";
+import { BadgeNames, ItemAvatar } from "@/components/Item";
+import { ColorTheme } from "@/theme/colors";
+import { FlexLayouts, Layouts } from "@/theme/layouts";
+import { Typography } from "@/theme/typography";
+import React from "react";
+import { SafeAreaView, View, Text, StyleSheet } from "react-native";
+import Octicons from "@expo/vector-icons/Octicons";
+import { mockCredentials } from "@/services/mocks";
+
+const AddCredentialDetailScreen = ({ navigation, route }) => {
+  const credential = route.params.credential;
+  const navigateToAddCredentialOptions = () => {
+    navigation.navigate("AddCredentialOptions", { credential });
+  };
+  const navigateToAddCredentialDetail = (secondaryCredential) => {
+    navigation.push("AddCredentialDetail", { credential: secondaryCredential });
+  };
+
+  //TODO: Replace mockCredentials with real profile credentials
+
+  return (
+    <SafeAreaView>
+      <View style={Layouts.container}>
+        <View style={[Layouts.row, FlexLayouts.containerHorizontalCenter]}>
+          <ItemAvatar
+            iconName="note"
+            badgeName={BadgeNames.CREDENTIAL}
+            style={Typography.textCenter}
+          />
+          <Text style={[Typography.heading2, Typography.textCenter]}>
+            {credential.name}
+          </Text>
+          <Text style={[Typography.body3, Typography.textCenter]}>
+            Issued by {credential.issuer}
+          </Text>
+        </View>
+        <View style={Layouts.row}>
+          <Text style={Typography.body4}>{credential.description}</Text>
+        </View>
+        <View style={[Layouts.row, Checklist.callout, Checklist.danger]}>
```

Comment:
This approach doesn't scale well. You can actually reuse your styles inside of each other outside of the component rather than compose arrays. For example:

```
const OtherStyles = StyleSheet.create({ thing: bar })
const CalloutStyles = StyleSheet.create({ danger:  [{ color: abc}, OtherStyles.thing] })
```

Reply:
So I would expect the following:

```js
const styles = StyleSheet.create({
 credentialsRequiredRow: [ Callouts.Thing, Callouts.Danger ]
});

<View style={styles.credentialsRequiredRow>
```

This has semantic meaning, performs better, scales better. If we are going to lean into composition then we can just go all out and do Tailwind